### PR TITLE
Updating Exteding Docker with PVC case

### PR DESCRIPTION
**Pull request by jayasimha-raghavan-unskript**
## Description
Updated Extending Docker document with the case when used in K8S and PVC attached to the POD.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/544964f9-b3c4-4f71-a6ce-7ab00ffb08a6

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/18c70d98-76dc-43c6-8eab-06505534b6c2

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/bf5faaf8-2985-4902-9b79-8cd0f0c56c02

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/43546124-9b49-40d1-ae02-8f6740d0008a

#### Extending with Awesome-CloudOps being the submodule

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/e3a0b79c-e5e2-4fc2-9793-591cc346b9f1

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: README_extending_docker.md [hunk 1]
```diff
@@ -9,41 +9,109 @@
...
+your custom Actions and/or runbooks and build a custom Docker image.
+1. 
+    1. Here we assume $HOME/Workspace is where you have your Git repo cloned
        ```
-       cd $ACA_ROOTDIR
-       mkdir custom
+       cd $HOME/Workspace  # Please change this directory whereever your Git repo is cloned
+       export $ACA_ROOTDIR=$PWD
        ```
     2. Submodule your Git repo that has your custom Actions in it.
```
Comment (amit-chandak-unskript) [inline README_extending_docker.md:96]: This statement needs to be updated.

File: README_extending_docker.md [hunk 2]
```diff
@@ -82,15 +59,83 @@ You can use our base docker to extend the functionality to fit your need. The st
...
+export CONTAINER_ID=`docker ps | grep awesome-runbooks | awk '{print $1}'`
+docker cp $CONTAINER_ID:/unskript/data/actions $HOME/Workspace/acme/actions
+docker cp $CONTAINER_ID:/unskript/data/runbooks $HOME/Workspace/acme/runbooks
+```
+
+Lets consider two scenarios as starting point for extending Awesome docker
+
+### How to create Custom Actions
+
+#### Steps for awesome-runbooks deployed as K8S POD
```
Comment (amit-chandak-unskript) [inline README_extending_docker.md:97]: Create custom actions should have been covered somewhere already, right? if not, just mention the export functionality, doesnt matter how it was launched

Comment (jayasimha-raghavan-unskript) [reply README_extending_docker.md]: Done

File: README_extending_docker.md (hunk 1 above)
Comment (jayasimha-raghavan-unskript) [reply README_extending_docker.md]: Reworded as per our discussion

**Review by amit-chandak-unskript — APPROVED** [review]

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**